en_US/changes/upgrade-4.4.md

```diff
@@ -0,0 +1,104 @@
+# Upgrade from 4.3 to 4.4
+
+EMQ X 4.3 and 4.4 nodes co-exist in the same cluster.
+Hence it's possible to apply a rolling upgrade of a EMQ X cluster.
+
+The upgrade steps of each EMQ X node are summarised below:
+
+1. Optional: remove the node from load-balancer
+1. Stop the node, (e.g. `emqx stop`, `systemctl stop emqx`)
+1. Backup 'data' and 'etc' directories. (more details below)
+1. Uninstall v4.3
+1. Install v4.4, and restore the backup directories for 'data' and 'etc'
+1. Start v4.4, and monitor alerts/error logs
+1. Optional: add the node back to the load-balancer
+
+NOTE: Mind the [Config changes](#config-change) and [Behaviour changes](#behaviour-changes).
```

Comment:
Nit: these links are not working (at least in the Github visualization)

Reply:
it's been fixed in the next PR.